src/gui/widgets/AutomatableSlider.cpp

```diff
@@ -74,7 +74,7 @@ void AutomatableSlider::mousePressEvent( QMouseEvent * _me )
 	}
 	else
 	{
-		IntModelView::mousePressEvent( _me );
+		IntModelView::mousePressEvent(_me);
```

Comment:
Changes to `AutomatableSlider.h` and `AutomatableSlider.cpp` could be reverted completely

Reply:
ok. reverted